app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
```

Comment:
`URI.join` would be great for the purpose, but the problem with it is that it always expect that the result is absolute URL. Relative URLs throw error:

```
2.3.1 :005 > URI.join("foo", "/bar", "baz", "/quaax")
URI::BadURIError: both URI are relative
```